File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/Microformat.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class Microformat{

	@SerializedName("playerMicroformatRenderer")
	private PlayerMicroformatRenderer playerMicroformatRenderer;

	public void setPlayerMicroformatRenderer(PlayerMicroformatRenderer playerMicroformatRenderer){
		this.playerMicroformatRenderer = playerMicroformatRenderer;
	}

	public PlayerMicroformatRenderer getPlayerMicroformatRenderer(){
		return playerMicroformatRenderer;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/Microformat.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class Microformat : Parcelable, Serializable {
    @SerializedName("playerMicroformatRenderer")
    var playerMicroformatRenderer: PlayerMicroformatRenderer? = null
    override fun toString(): String {
        return "Microformat{" +
                "playerMicroformatRenderer = '" + playerMicroformatRenderer + '\'' +
                "}"
    }
}